### PR TITLE
Fix CDCCommunityLevelMetrics scraper.

### PR DESCRIPTION
**Pull request by mikelehen**

### CODE REVIEW

File: can_tools/scrapers/official/federal/CDC/cdc_community_level.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from can_tools.scrapers.official.base import FederalDashboard
 
 
-COMMUNITY_LEVEL_MAP = {"Low": 0, "Medium": 1, "High": 2}
+COMMUNITY_LEVEL_MAP = {"Low": 0, "Medium": 1, "High": 2, "low": 0, "medium": 1, "high": 2}
```
Comment (mikelehen) [inline can_tools/scrapers/official/federal/CDC/cdc_community_level.py:6]: The lowercase versions are only used for a few places (Guam, America Samoa, etc.).  I wouldn't be surprised if it's a mistake and ends up getting fixed. 🤷

File: can_tools/scrapers/official/federal/CDC/cdc_community_level.py [hunk 2]
```diff
@@ -46,10 +46,9 @@ def fetch(self):
 
     def normalize(self, data: pd.DataFrame) -> pd.DataFrame:
 
-        # Transform percentages to ratios (e.g 3% -> 0.03)
+        # Transform percentages to ratios (e.g 3 -> 0.03)
         data["covid_inpatient_bed_utilization"] = (
-            pd.to_numeric(data["covid_inpatient_bed_utilization"].str.replace("%", ""))
-            / 100
+            data["covid_inpatient_bed_utilization"] / 100
```
Comment (mikelehen) [inline can_tools/scrapers/official/federal/CDC/cdc_community_level.py:51]: Per https://data.cdc.gov/Public-Health-Surveillance/United-States-COVID-19-Community-Levels-by-County/3nnm-4jni 
> March 31, 2022: The “%” sign was removed from the text field in column “covid_inpatient_bed_utilization”. No change was made to the data. As indicated in the column description, values in this column represent the percentage of staffed inpatient beds occupied by COVID-19 patients (7-day average).

File: can_tools/scrapers/official/federal/CDC/cdc_community_level.py [hunk 3]
```diff
@@ -62,11 +61,6 @@ def normalize(self, data: pd.DataFrame) -> pd.DataFrame:
                 date_column="date_updated",
                 # Remove state-equivalent territory locations
                 locations_to_drop=[
-                    "000AS",
-                    "000GU",
-                    "000VI",
-                    "000MP",
-                    "UNK",
```
Comment (mikelehen) [inline can_tools/scrapers/official/federal/CDC/cdc_community_level.py:69]: Per https://data.cdc.gov/Public-Health-Surveillance/United-States-COVID-19-Community-Levels-by-County/3nnm-4jni

> March 31, 2022: FIPS codes for territories American Samoa, Guam, Commonwealth of the Northern Mariana Islands, and United States Virgin Islands were re-formatted to 5-digit numeric for records released on 3/3/2022 to be consistent with other records in the dataset.

File: can_tools/scrapers/official/federal/CDC/cdc_community_level.py [hunk 4]
```diff
@@ -3,7 +3,14 @@
...
 
 
-COMMUNITY_LEVEL_MAP = {"Low": 0, "Medium": 1, "High": 2}
+COMMUNITY_LEVEL_MAP = {
+    "Low": 0,
+    "Medium": 1,
+    "High": 2,
+    "low": 0,
+    "medium": 1,
+    "high": 2,
```
Comment (mikelehen) [inline can_tools/scrapers/official/federal/CDC/cdc_community_level.py:12]: The lowercase versions are only used for a few places (Guam, America Samoa, etc.). I wouldn't be surprised if it's a mistake and ends up getting fixed. 🤷

**Review by smcclure17 — APPROVED** [review]
Awesome, looks good to me! thanks for fixing this

**Pull request merged**